app/Http/Controllers/DashboardController.php

```diff
@@ -5,10 +5,64 @@
 namespace App\Http\Controllers;
 
 use Inertia\Inertia;
+use App\Models\User;
+use App\Models\Exam;
+use App\Models\Module;
+use App\Models\Task;
+use App\Models\Event;
+use Illuminate\Http\Request;
 
-class DashboardController extends Controller
-{
+class DashboardController extends Controller {
     public function index() {
-        return Inertia::render('Dashboard');
+        $user_id = User::query()->where('id', '=', 1)->get()[0]->id;
+        $modules = Module::query()->where('user_id', '=', $user_id)->get();
+        $tasks = Task::query()->where('user_id', '=', $user_id)->get();
+        $events = Event::query()
+            ->where('user_id', $user_id)
+            ->whereDate('start', '>=', now()->toDateString())
```

Comment:
Würde den Zeitraum auf 1 Woche erweitern.

Zusätzlich sollte man schauen ob es laufende events gibt, also start in der Vergangenheit / ende in der Zukunft. Z.b.: wenn man sich eine Einführungswoche einträgt als Mehrtagesevent.